test_utils/mixins.py

```diff
@@ -0,0 +1,50 @@
+"""
+Mixins for program-intent-engagement API tests.
+"""
+from time import time
+
+import jwt
+from django.conf import settings
+
+JWT_AUTH = "JWT_AUTH"
+
+
+class JwtMixin:
```

Comment:
I'd prefer to just put this in the API test case which is its only user. Right now you have to read three files to know what is going on - the actual test, the api test case class, and this file

we can always break things apart for reuse later when there is actual reuse